FAQ.md

```diff
@@ -0,0 +1,66 @@
+# Frequently Asked Questions
+
+## Do Angular CDK and Angular Material support Shadow DOM?
+
+While we don't _officially_ support Shadow DOM, we make a "best effort" to keep the components
+working in applications that do use Shadow DOM. This may change in the future based on the evolving
+browser landscape.
+
+## What do the project labels mean?
+
+Some labels are hopefully obvious, such as "feature" and "a11y". Other issues should all have
+descriptions on GitHub that outline how they're used. See our [labels page][labels] for the full
+descriptions.
+
+## Are there any updates on issue _X_?
+
+Any issue updates will appear on the issue. Due to the large volume of issues and feature requests
+recieved by the team, we aren't able to regularly comment on all open issues and PRs.
+
+## Why hasn't PR _X_ been merged?
+
+For every pull request, we run a presubmit against Google's internal test suite. This includes tests
+for all of the projects that use Angular CDK and Angular Material inside Google. As you might
+imagine, this process can be slow. Once this presubmit passes, PRs can generally be merged quickly.
+When tests fail, however, the team has to spend time investigating before the PR can be merged.
+[Google uses a single monolithic code repository for its code, which everything at head][monorepo].
+Because of this, we cannot merge any PRs that would break an existing project. If a PR has extensive
+failures, it may be put on the backburner until the team can schedule time to debug the full extent
+of the issue. If a PR seems ready to merge, but has been inactive, it has very likely
+encountered some test failures inside Google that must be resovled first.
+
+## Why aren't you working on _X_?
+
+Like any software team, we have limited time and resources. On top of the work we do in this repo,
+our team builds and maintains a smaller suite of Google-internal UI components and provides support
+to product teams inside Google using our components. We do our best to balance our time between bug
```

Comment:
I wonder if it would be good to note the scale of these component's use within Google. I think that people would appreciate this more if they knew that this meant supporting x-hundred or x-thousand different teams. Obviously the number would be rough, but giving some idea of the scale might be helpful.